ishimwe5555/AnnounceIT
# #170864382 signup-api-endpoints

Description
Created API Endpoint that uses verb/request POST to add data everytime a user creates an account 
 Type of change
Please select the relevant option
- [v] New feature (non-breaking change which adds functionality)
How Has This Been Tested?
- [v] Unit
 How to Test
Here are instructions on how reviewers can verify my work
-Clone the repository
-Using a terminal,run npm i then npm start to start the node server
-Using postman,turn header key to "Content-type" and value to "Application/Json" 
-Do the same in body too
-Then send a post request with the following parameters and values respectively,you should see a message that says "User created!" and you can see the values entered in console

               {
                   “id” : Integer,
                   “email” : String,
                   “first_name” : String,
                   “last_name” : String,
                    “password” : String,
                   “phoneNumber” : String,
                   “address” : String,
                   “is_admin” : Boolean,

Any background context you want to add
VSCode,Postman and Node.js command prompt for better use
Pivotal Tracker
https://www.pivotaltracker.com/story/show/170864382

[Finishes  #170864382]
